docs/ds/code/top-tree/top-tree_1.cpp

```diff
@@ -0,0 +1,189 @@
+#include <bits/stdc++.h>
+#define ls(x) T[x][0]
+#define rs(x) T[x][1]
+#define ms(x) T[x][2]
+using namespace std;
+
+const int maxn = 300005;
```

Comment:
It may be better to use capital characters for constants.

Reply:
done